node/src/gql/types/answer.ts

```diff
@@ -23,6 +22,7 @@ import {
   ExternalVideoIdsObjectType,
   externalVideoIdsDefault,
 } from '../mutation/api/update-answers';
+import { dataLoaders } from '../../lambda';
```

Comment:
why is this located in the lambda folder?

Reply:
this is not specific to the lambda and consequently should not be directly located in the lambda function itself.  How cpu intensive is the creation of the dataloaders?  If it isn't too bad you may want to create them dynamically instead.  Alternatively you could use a singleton.  In this case I don't think that would be improper.

Reply:
Good call, changed to load it dynamically and provide as context per request, per the packages readme: "Typically, DataLoader instances are created when a Request begins, and are not used once the Request ends."